app/javascript/shared/track/crisp.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
+const { key, enabled, administrateur } = gon.crisp || {};
+
+if (enabled) {
+  window.$crisp = [];
+  window.CRISP_WEBSITE_ID = key;
+  (function() {
+    d = document;
+    s = d.createElement('script');
+    s.src = 'https://client.crisp.chat/l.js';
+    s.async = 1;
+    d.getElementsByTagName('head')[0].appendChild(s);
+  })();
+  $crisp.push(['set', 'user:email', [administrateur.email]]);
+  $crisp.push(['set', 'session:segments', [['administrateur']]]);
+  $crisp.push(['set', 'session:data', [[['DS_ID', administrateur.email],['DS_SIGN_IN_COUNT', administrateur.DS_SIGN_IN_COUNT],['URL_MANAGER', 'https://www.demarches-simplifiees.fr/manager/administrateurs/'+administrateur.DS_ID]]]]);
```

Comment:
Pinaillage : éventuellement ça peut être plus lisible en mettant un élément par ligne.

```suggestion
  $crisp.push(
    ['set', 'session:data', [[
	  ['DS_ID', administrateur.email],
	  ['DS_SIGN_IN_COUNT', administrateur.DS_SIGN_IN_COUNT],
	  ['URL_MANAGER', 'https://www.demarches-simplifiees.fr/manager/administrateurs/'+administrateur.DS_ID]
	]
  ]]);
```

(D'ailleurs y'a pas un niveau de tableau de trop ?)

Reply:
(non, pas de niveau de tableau en trop)